dartagnan/src/test/java/com/dat3m/dartagnan/AbstractSvCompTest.java

```diff
@@ -34,15 +36,14 @@
     private final Wmm wmm;
     private final Settings settings;
     private Result expected;
-    
+
```

Comment:
We should leave another of the base tests run. Currently we only run refinement on svcomp benchmarks.

Reply:
Sure, I will enable the combined tests

Reply:
There is no combined test for `AbstractSvCompTest`. I think the following configuration should be enough to find most potential bugs

- `AbstractDartagnanTest`: run both combined (test refinement and two solvers on modified reach conditions) and two solvers (our usual test suit)
- `AbstractSvCompTest`: run incremental (fast and would probably catch most bugs in the backend related to some optimisations)
- `SafeCTest`: run refinement (tests the compilation schemes and refinement)

By the way ... most of the code of our test is completely repeated except by the analysis method called. Do you know if there is any way to parametrise this?

Reply:
Oh, I thought it was about the Litmus tests... The incremental solver did run for most past tests. I just disabled it for some local tests and forgot to enable it again. I will enable them again